README.md

```diff
@@ -40,50 +40,59 @@ Used to validate mutect2 workflow.
 - GATK4 or later 
 
 Cromwell version support 
-- Successfully tested on v31
+- Successfully tested on v36
 
 
 ### Parameter descriptions :
 #### mutect2 (single pair/sample)  
-- ``Mutect2.gatk4_jar`` -- Please see parameter description above in the mutect2_multi_sample.
-- ``Mutect2.intervals`` -- Please see parameter description above in the mutect2_multi_sample.
-- ``Mutect2.ref_fasta`` -- Please see parameter description above in the mutect2_multi_sample.
-- ``Mutect2.ref_fasta_index`` -- Please see parameter description above in the mutect2_multi_sample.
-- ``Mutect2.ref_dict`` -- Please see parameter description above in the mutect2_multi_sample.
+- ``Mutect2.gatk4_jar`` -- Location *within the docker file* of the GATK4 jar file.  If you wish you to use a different jar file, such as one on your local filesystem or a google bucket, specify that location with ``Mutect2_Multi.gatk4_jar_override``.  This parameter is ignored if ``Mutect2_Multi.gatk4_jar_override`` is specified.
+- ``Mutect2.intervals`` -- A file listing genomic intervals to search for somatic mutations.  This should be in the standard GATK4 format.
+- ``Mutect2.ref_fasta`` 	-- reference fasta.  For Broad internal VM:  ``/seq/references/Homo_sapiens_assembly19/v1/Homo_sapiens_assembly19.fasta``
```

Comment:
I'm not sure how this example json is used, but it could be more helpful to point to the gatk public resource buckets than to paths on the Broad servers.

Reply:
Ha, that would be a really good idea. Good catch